lib/client/src/context.rs

```diff
@@ -156,6 +164,8 @@ pub struct MangoGroupContext {
     pub address_lookup_tables: Vec<Pubkey>,
 
     pub compute_estimates: ComputeEstimates,
+
+    pub fallback_oracle_context: FallbackOracleContext,
```

Comment:
I'm doubtful of putting this user-side configuration into the MangoGroupContext - its intention is to cache the on chain state that's needed frequently and changes rarely.